thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/DetectionPipelineResult.java

```diff
@@ -31,6 +32,7 @@
 
   Map<String, Object> diagnostics;
   List<MergedAnomalyResultDTO> anomalies;
+  TimeSeries predictions;
```

Comment:
Need a Map from string to TimeSeries here. We need a predicted time series per detection rule and dimension value.